src/com/lftechnology/java/training/alina/basic/smallestnumber/SmallestPositiveNumber.java

```diff
@@ -5,74 +5,69 @@
 import java.util.logging.Logger;
 
 /**
- * 2520 is the smallest number that can be divided by each of the numbers from 1
- * to 10 without any remainder. What is the smallest positive number that is
- * evenly divisible by all of the numbers from 1 to 20?
+ * 2520 is the smallest number that can be divided by each of the numbers from 1 to 10 without any remainder. What is the smallest positive
+ * number that is evenly divisible by all of the numbers from 1 to 20?
  * 
  * @author Alina Shakya <alinashakya@lftechnology.com>
  *
  */
 public class SmallestPositiveNumber {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)